benchmarks/src/test/java/io/crate/data/join/RowsBatchIteratorBenchmark.java

```diff
@@ -191,4 +212,25 @@ public void measureConsumeHashInnerJoinWithHashCollisions(Blackhole blackhole) {
             blackhole.consume(leftJoin.currentElement().get(0));
         }
     }
+
+    @Benchmark
+    public void measureConsumeWindowBatchIterator(Blackhole blackhole) {
+        InputCollectExpression input = new InputCollectExpression(0);
```

Comment:
this can be in the setup method as well

Reply:
as discussed, left is as it is as it is tightly coupled to the test